### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -3,7 +3,9 @@
     to Adrien Nayrat for the report)
   - Don't try to detect if a hypotetical index would be used if no suitable
     index is detected (Julien Rouhaud, thanks to Guillaume Lelarge for the
-    repot)
+    report)
+  - Fix lost filter when changing the time range (thanks to Marc Cousin for
+    the report)
```
Comment (rjuju) [inline CHANGELOG]: please add your name as author of the fix :)

**Comment by pgiraud** [comment]
@rjuju please review again

File: CHANGELOG [hunk 2]
```diff
@@ -3,7 +3,10 @@
     to Adrien Nayrat for the report)
   - Don't try to detect if a hypotetical index would be used if no suitable
     index is detected (Julien Rouhaud, thanks to Guillaume Lelarge for the
-    repot)
+    report)
+  - Fix lost filter when changing the time range (Pierre Giraud, thanks to Marc
+    Cousin for the report)
+    report)
```
Comment (rjuju) [inline CHANGELOG]: missed conflict during rebase?

Comment (pgiraud) [reply CHANGELOG]: You caught me!

**Comment by rjuju** [comment]
Thanks!

**Pull request merged**